CMakeLists.txt

```diff
@@ -3,10 +3,20 @@ cmake_minimum_required (VERSION 3.0)
 
 project(SWMM)
 
-set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
-set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR}/lib)
+#set(EXECUTABLE_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/bin)
+#set(LIBRARY_OUTPUT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/lib)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+if(MSVC)
+    set(CMAKE_C_FLAGS "/fp:fast /fp:except- /Gd /Oy- /Ob2 /Ot")
+    set(CMAKE_C_FLAGS_DEBUG "/MTd /W4")
+    set(CMAKE_C_FLAGS_RELEASE "/O2 /MT /W3")
+
+    set(CMAKE_CXX_FLAGS "/fp:fast /fp:except- /Gd /Oy- /Ob2 /Ot")
+    set(CMAKE_CXX_FLAGS_DEBUG "/MTd /W4")
+    set(CMAKE_CXX_FLAGS_RELEASE "/O2 /MT /W3")
```

Comment:
@bemcdonnell it would be nice to add some "docs" or comments around what the flags mean :-)